scripts/x-charts.exports.json

```diff
@@ -157,14 +170,12 @@
   { "name": "getChartsTooltipUtilityClass", "kind": "Function" },
   { "name": "getGaugeUtilityClass", "kind": "Function" },
   { "name": "getHighlightElementUtilityClass", "kind": "Function" },
-  { "name": "getLegendUtilityClass", "kind": "Function" },
   { "name": "getLineElementUtilityClass", "kind": "Function" },
   { "name": "getMarkElementUtilityClass", "kind": "Function" },
   { "name": "getPieArcLabelUtilityClass", "kind": "Function" },
   { "name": "getPieArcUtilityClass", "kind": "Function" },
   { "name": "getPieCoordinates", "kind": "Function" },
   { "name": "getReferenceLineUtilityClass", "kind": "Function" },
-  { "name": "getSeriesToDisplay", "kind": "Function" },
```

Comment:
Could add in the list of breaking changes that this function got replaced by `useLegend`